### DISCUSSION TITLE
water damage

### DISCUSSION THREAD

**Discussion by MrTokyo12345** [BG-Software-LLC/SSB-AcidIslands#6] [Ideas]
Possible to add in the acidisland modules the possibility of giving the poison effect when a player enters the water.

**Comment by OmerBenGera** [comment]
Players take damage when they enter water already

**Comment by MrTokyo12345** [comment]
I would need that from poison damage would it be possible?

**Comment by balajin96** [comment]
Entering into water, player takes damage.
